src/Microsoft.ML.AutoML/Tuner/PipelineProposer.cs

```diff
@@ -119,11 +121,12 @@ public PipelineProposer(SweepablePipeline sweepablePipeline, AutoMLExperimentSet
                 schema = _pipelineSchemas[i];
             }
 
-            return (_sweepablePipeline.BuildSweepableEstimatorPipeline(schema).SearchSpace, schema);
+            return schema;
         }
 
         public void Update(TrialResult result, string schema)
         {
+            _initialized = true;
```

Comment:
>_initialized [](http://example.com/codeflow?start=12&length=12)

should you set this after finishing the update?

Reply:
### Resolved